src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableList_1.cs

```diff
@@ -16,7 +14,7 @@ namespace System.Collections.Immutable
     /// </summary>
     /// <typeparam name="T">The type of elements in the set.</typeparam>
     [DebuggerDisplay("Count = {Count}")]
-    [DebuggerTypeProxy(typeof(ImmutableListDebuggerProxy<>))]
+    [DebuggerTypeProxy(typeof(ImmutableEnumerableDebuggerProxy<>))]
```

Comment:
@AArnott I finally managed to get a local build of S.C.I.dll running with a console app. Here's visual proof it works:

<img width="885" alt="imlist_debug" src="https://user-images.githubusercontent.com/9159214/28906076-ff8261d8-77e2-11e7-8ec4-943c91de42a2.PNG">